src/wizard/unreal/index.md

```diff
@@ -25,11 +25,69 @@ Access the Sentry configuration window by going to editor's menu: `Project Setti
 ___PUBLIC_DSN___
 ```
 
-<Alert level="warning" title="Warning">
+## Crash Report Client
 
 For Windows and Mac [Crash Report Client](/platforms/unreal/setup-crashreport/) provided along with Unreal Engine has to be configured in order to capture errors automatically.
 
-</Alert>
+### Include the UE4 Crash Reporter
+
+You can add the _Crash Report Client_ to your game in your _Project Settings_.
```

Comment:
Is Project Settings a page in the configuration editor or some other UI element below the page level?

Reply:
Here `Project Settings` is an Unreal Engine editor window used for project/plugin configuration.